include/simple/server.hpp

```diff
@@ -132,11 +133,11 @@ class Server {
    * callback function and reply.
    */
   void awaitRequest(std::shared_ptr<std::atomic<bool>> alive, std::shared_ptr<GenericSocket<T>> socket) {
-    while (*alive) {
+    while (alive->load()) {
```

Comment:
The previous version was as good, but I guess you were going for a consistent use of load()/store() for all atomics?

Reply:
yes, just a matter of consistency